### PR TITLE
Fixed mobject gallery and class regex matching

### PR DESCRIPTION
**Pull request by KekOnTheWorld**

### CODE REVIEW

**Comment by KekOnTheWorld** [comment]
Fixing eslint warnings rn

File: src/gallery.ts [hunk 1]
```diff
@@ -212,56 +207,61 @@ export class Gallery {
...
-          );
-          return;
-        }
+    var {data} = await axios.get(GITHUB_ENTRY_FILE);
+    data = yaml.load(data) as any; // Parse yaml
+    
+    let newVersion = data["user_content_version"];
+    
+    // Related to https://github.com/kolibril13/mobject-gallery/issues/3
+    const galleryParameters =
```
Comment (Rickaym) [inline src/gallery.ts:216]:
```suggestion
    const galleryParameters = https://github.com/kolibril13/mobject-gallery/issues/3
```

File: src/gallery.ts [hunk 2]
```diff
@@ -212,56 +207,61 @@ export class Gallery {
...
-        }
+    var {data} = await axios.get(GITHUB_ENTRY_FILE);
+    data = yaml.load(data) as any; // Parse yaml
+    
+    let newVersion = data["user_content_version"];
+    
+    // Related to https://github.com/kolibril13/mobject-gallery/issues/3
+    const galleryParameters = https://github.com/kolibril13/mobject-gallery/issues/3
+      data["gallery_parameters_path"] ||
+      data["gallary_parameters_path"];
```
Comment (Rickaym) [inline src/gallery.ts:218]:
```suggestion
    const galleryParameters = data["gallery_parameters_path"];
```
Issue seems to be fixed before the merge!

**Review by Rickaym — APPROVED** [review]
Great PR, thank you very much keky.

**Pull request merged**